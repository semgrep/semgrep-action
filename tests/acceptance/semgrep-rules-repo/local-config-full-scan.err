=== Running: SEMGREP_RULES="python/lang/best-practice/sleep.yaml" semgrep ci --no-suppress-errors
Running `semgrep ci` without API token but with configs ('python/lang/best-practice/sleep.yaml',)


╭────────────────╮
│ Debugging Info │
╰────────────────╯

  SCAN ENVIRONMENT
  versions    - semgrep x.y.z on python x.y.z
  environment - running in environment git, triggering event is unknown


╭─────────────╮
│ Scan Status │
╰─────────────╯

Fetching configuration from semgrep.dev
Scanning 210 files.


╭─────────╮
│ Results │
╰─────────╯



╭──────────────╮
│ Scan Summary │
╰──────────────╯

Some files were skipped or only partially analyzed.
  Scan was limited to files tracked by git.
  Scan skipped: 5 files matching .semgrepignore patterns
  For a full list of skipped files, run semgrep with the --verbose flag.

CI scan completed successfully.
  Found 2 findings (2 blocking) from 1 rule.
  Has findings for blocking rules so exiting with code 1
