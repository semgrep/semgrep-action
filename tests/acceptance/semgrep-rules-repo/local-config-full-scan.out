
Blocking Findings:

  python/lang/best-practice/sleep.py
     python.lang.best-practice.arbitrary-sleep
        time.sleep() call; did you mean to leave this in?

         12┆ t.sleep(5)
          ⋮┆----------------------------------------
         14┆ t.sleep(0.1)

Blocking Rules Fired:
   python.lang.best-practice.arbitrary-sleep
